features/support/env.rb

```diff
@@ -19,7 +18,7 @@
 
 After do |_scenario|
   if File.exist?(File.join(aruba.config.working_directory, 'Vagrantfile'))
-    Komenda.run('bundle exec vagrant destroy -f', cwd: aruba.config.working_directory, fail_on_fail: true)
+    #Komenda.run('bundle exec vagrant destroy -f', cwd: aruba.config.working_directory, fail_on_fail: true)
```

Comment:
watch out here. You commented out some test harness setup/teardown stuff and accidentally committed it.